### PR TITLE
osx/*: update page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

# ToDo

- [x] Add missing man page's links. There are no man pages found for these commands:
  - [ ] applecamerad
  - [ ] archey
  - [ ] timed
- [x] Prefer long options over short ones.
- [x] Fix quoting according to #7521.
- [x] Reformat one-liners code for awk, sed and etc.
- [x] Move `--help`|`--version` note to the top not to pollute examples

### CODE REVIEW

File: pages/osx/airport.md
Comment (navarroaxel) [inline pages/osx/airport.md:16]: what about that `=`?

File: pages/osx/base64.md
Comment (navarroaxel) [inline pages/osx/base64.md:20]: I think a base64 text doesn't need `"`.

File: pages/osx/bless.md
Comment (navarroaxel) [inline pages/osx/bless.md:16]: this doesn't need `"`.

File: pages/osx/brightness.md
Comment (navarroaxel) [inline pages/osx/brightness.md:4]: Just use the repo
```suggestion
> More information: <https://github.com/nriley/brightness>.
```

File: pages/osx/nvram.md
Comment (navarroaxel) [inline pages/osx/nvram.md:16]: are you sure?

File: pages/osx/ping.md
Comment (navarroaxel) [inline pages/osx/ping.md:8]:
```suggestion
`ping "{{hostname}}"`
```

File: pages/osx/route.md
Comment (navarroaxel) [inline pages/osx/route.md:9]:
```suggestion
`sudo route add "{{destination_ip_address}}" "{{gateway_address}}"`
```

File: pages/osx/route.md
Comment (navarroaxel) [inline pages/osx/route.md:17]:
```suggestion
`sudo route -t add "{{destination_ip_address}}/24" "{{gateway_address}}"`
```

File: pages/osx/route.md
Comment (navarroaxel) [inline pages/osx/route.md:25]:
```suggestion
`sudo route delete "{{destination_ip_address}}/24"`
```

File: pages/osx/rubocop.md
Comment (navarroaxel) [inline pages/osx/rubocop.md:4]: I think just the URL is fine.
```suggestion
> More information: <https://docs.rubocop.org/rubocop/usage/basic_usage.html>.
```

File: pages/osx/sed.md
Comment (navarroaxel) [inline pages/osx/sed.md:8]: I think the `'` is to avoid issues with escape sequences or env vars.

File: pages/osx/shutdown.md
Comment (navarroaxel) [inline pages/osx/shutdown.md:20]: I think is not required for this specific value
```suggestion
`shutdown -r +{{5}}`
```

File: pages/osx/whence.md
Comment (navarroaxel) [inline pages/osx/whence.md:4]: Not Oracle please.
```suggestion
> More information: <https://www.unix.com/man-page/OpenSolaris/1/whence/>.
```

File: pages/osx/airport.md
Comment (EmilySeville7cfg) [reply pages/osx/airport.md:16]: It is [intended](https://ss64.com/osx/airport.html):

```
      -s[arg]
      --scan=[arg]     Perform a wireless broadcast scan.
                       Will perform a directed scan if the optional arg is provided.
```

as I figured out from docs.

File: pages/osx/base64.md
Comment (EmilySeville7cfg) [reply pages/osx/base64.md:20]: Potentially this placeholder can contain text with spaces that's why I quoted it to ensure that no word splitting is performed.

File: pages/osx/sed.md
Comment (EmilySeville7cfg) [reply pages/osx/sed.md:8]: Maybe it makes sense to use single quotes for sed/awk and other inlined code. :)

File: pages/osx/nvram.md
Comment (EmilySeville7cfg) [reply pages/osx/nvram.md:16]:
```bash
echo 1"2"
echo "12"
```
both output `12`. That's why I moved one quote left.

File: pages/osx/ping.md
Comment (EmilySeville7cfg) [reply pages/osx/ping.md]: Is it too verbose?

File: pages/osx/nvram.md
Comment (EmilySeville7cfg) [reply pages/osx/nvram.md:16]: On the other side quote after `=` looks more natural. :thinking:

File: pages/osx/airport.md
Comment (navarroaxel) [reply pages/osx/airport.md:16]: I'm sure is not required, if no `arg` is provided
```suggestion
`airport --scan`
```

File: pages/osx/rubocop.md
Comment (navarroaxel) [inline pages/osx/rubocop.md:4]: This page is not specific of `osx`, must be in `common/` platform.

File: pages/osx/ping.md
Comment (navarroaxel) [inline pages/osx/ping.md:8]: I think we should use `hostname`
```suggestion
`ping "{{hostname}}"`
```

File: pages/osx/base64.md
Comment (navarroaxel) [reply pages/osx/base64.md:20]: a base64 string doesn't have spaces.

Comment (EmilySeville7cfg) [reply pages/osx/base64.md:21]: Thanks. It is my mistake. :)

**Comment by EmilySeville7cfg** [comment]
Please update #6062 after merging PR. :)

**Review by blueskyson — COMMENTED** [review]
So many pages use 
```
> Get help or version: rubocop --help|--version.
```
or
```
> There are no options to get help or version.
```
But our style-guide doesn't suggest people add examples or version information in `>` lines.

I personally disapprove this because people can easily try `-h`, `--help`, `-v`, `--version` to get the information. If we accept this format, we should move `--help`, `--version` examples into `> get version ...` of other pages. What do you think about it?
@marchersimon @CleanMachine1 @navarroaxel

**Comment by EmilySeville7cfg** [comment]
@blueskyson, I've reverted my commit introducing `--help`|`--version` notes at the top of pages.

**Review by blueskyson — APPROVED** [review]
Thanks for your contribution. You can still open an issue for the `--help|--version` string. If several active maintainers accept this idea, then we can add `--help|--version` strings back.

File: pages/osx/dmesg.md
Comment (navarroaxel) [inline pages/osx/dmesg.md:4]:
```suggestion
> More information: <https://www.manpagez.com/man/8/dmesg/>.
```

File: pages/osx/open.md
Comment (navarroaxel) [inline pages/osx/open.md:4]:
```suggestion
> More information: <https://ss64.com/osx/open.html>.
```

File: pages/osx/rename.md
Comment (navarroaxel) [inline pages/osx/rename.md:4]:
```suggestion
> More information: <https://www.manpagez.com/man/2/rename/>.
```

File: pages/osx/wc.md
Comment (navarroaxel) [inline pages/osx/wc.md:8]: Revert this file. macOS doesn't support long arguments
```suggestion
`wc -l {{path/to/file}}`
```
Maybe we should use this link: https://ss64.com/osx/wc.html

File: pages/osx/oathtool.md
Comment (navarroaxel) [inline pages/osx/oathtool.md:4]: This is the author's page
```suggestion
> More information: <https://www.nongnu.org/oath-toolkit/oathtool.1.html>.
```

This page should be moved to `common/` because is available in Linux too.

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/osx/wc.md
Comment (EmilySeville7cfg) [reply pages/osx/wc.md]: Fixed ;)

File: pages/osx/oathtool.md
Comment (navarroaxel) [inline pages/osx/oathtool.md:1]: move this page to `common/`

**Pull request merged**
